migrations/script.py.mako

```diff
@@ -0,0 +1,24 @@
+"""${message}
```

Comment:
Never saw something like this...

Reply:
Like I wrote in the previous comment - it was done not by me (at least not by me manually)  (: